api/models/task.go

```diff
@@ -23,7 +23,7 @@ const (
 
 const (
 	// FormatDefault ...
-	FormatDefault = ""
+	FormatDefault = "default"
```

Comment:
I've searched for that constant and there's no other case.
I've just committed that change.